packages/auth/src/openapi/specs/id-provider.yaml

```diff
@@ -39,6 +39,16 @@ paths:
               description: Interaction id
         '401':
           description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/gnap-error'
+        '500':
+          description: Internal Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/gnap-error'
```

Comment:
I figured it might make sense to include this in the auth server spec as well for Open Payments, so I captured it in an issue [here](https://github.com/interledger/open-payments/issues/518).

Reply:
Makes sense! if you want, we could probably merge the open payments issue first, bump the client, and import the yaml files in here so we can reference them as `./auth-server.yaml#/components/schemas/...`, what do you think? Unless we want to decouple the IDP spec from the auth server spec?

Reply:
Let's bump the OP client for this :+1: